File: java/Program/Baju.java
public class Baju extends Aksesoris {
    private String untuk;
    private String size;
    private String merk;

    public Baju() {
        super();
        this.untuk = "";
        this.size = "";
        this.merk = "";
    }

    public Baju(int id, String name, int stock, double price, String jenis, String bahan, String warna, String untuk, String size, String merk) {
        super(id, name, stock, price, jenis, bahan, warna);
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    public void setUntuk(String untuk) { this.untuk = untuk; }
    public void setSize(String size) { this.size = size; }
    public void setMerk(String merk) { this.merk = merk; }

    public String getUntuk() { return this.untuk; }
    public String getSize() { return this.size; }
    public String getMerk() { return this.merk; }
}

File: python/Program/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
    def __init__(self, id=0, name="", stock=0, price=0.0, jenis="", bahan="", warna="", untuk="", size="", merk=""):
        super().__init__(id, name, stock, price, jenis, bahan, warna)
        self._untuk = untuk
        self._size = size
        self._merk = merk
    
    def set_untuk(self, untuk): self._untuk = untuk
    def get_untuk(self): return self._untuk
    
    def set_size(self, size): self._size = size
    def get_size(self): return self._size
    
    def set_merk(self, merk): self._merk = merk
    def get_merk(self): return self._merk
